github/orgs_teams.go

```diff
@@ -41,6 +44,19 @@ func (t Team) String() string {
 	return Stringify(t)
 }
 
+// Invitation represents a team member's inviation status
+type Invitation struct {
+	ID        *int       `json:"id,omitempty"`
+	Login     *string    `json:"login,omitempty"`
+	Email     *string    `json:"email,omitempty"`
+	Role      *string    `json:"role,omitempty"`
+	CreatedAt *time.Time `json:"created_at,omitempty"`
```

Comment:
@shurcooL 
FYI https://developer.github.com/changes/2016-12-20-organization-team-api-update/

You can see the github announced api change log.
So, maybe github will update api doc. :)